NhanVien.dart

```diff
@@ -1,14 +1,22 @@
-class NhanVien{
+abstract class NhanVien {
   static int maNVCount = 1;
   String maNV = '${maNVCount++}';
   String hoTen;
   String ngaySinh;
   String soDT;
   String diaChi;
   double luong;
-  NhanVien({required this.hoTen, required this.ngaySinh, required this.soDT, required this.diaChi, required this.luong});
+  double realSalary = 0;
```

Comment:
This property is not necessary anymore

Reply:
change to get property

Reply:
I removed `realSalary` and print salary by print `calculateSalary()`.